frontend/src/Order.tsx

```diff
@@ -17,11 +17,7 @@ import { FieldHeader } from './components/FieldHeader'
 import { WbsPicker } from './components/WbsPicker'
 import { ErrorHandler } from './components/ErrorHandler'
 
-interface Props {
-    topRef: React.RefObject<HTMLElement>
-}
-
-const Order = ({ topRef }: Props) => {
+const Order = () => {
```

Comment:
Maybe the Order component can have a prop like `isSideSheet` which defines whether the component is being used in the table view or in the main order view Then the `div`-style and the values for `xs` and `sm` can be set according to the value of `isSideSheet`

Reply:
Yes good idea, will implement

Reply:
Fixed